integrationTests/CreateDonationTest.php

```diff
@@ -49,23 +50,27 @@ public function testCannotCreateDonationWithNegativeTip(): void
         $this->assertSame(400, $response->getStatusCode());
     }
 
-    public function testITReturns400OnRequestToDonateNullAmount(): void
+    public function testItReturns400OnRequestToDonateNullAmount(): void
     {
         $campaignId = $this->setupNewCampaign();
 
         $response = $this->getApp()->handle(
             new ServerRequest(
-                'POST',
-                '/v1/donations',
+                method: 'POST',
+                uri: '/v1/people/12345678-1234-1234-1234-1234567890ab/donations',
```

Comment:
this string is repeated *a lot* of times, and it has to be the same every time. Can we replace it with a constant instead of adding new copies of it?

Reply:
Thanks - is there a reason for preferring a function rather than a constant though? IMHO constants are generally preferable because they're simpler, there's no need to look to see how it's implemented when you find a usage.

Reply:
No no strong preference either way here really, I just went with what was closest to the most similar helper

Reply:
OK not a blocker.